### PR TITLE
docs(`terraform_docs`): Clarify requrement of `--args=--config=`

### PR DESCRIPTION
**Pull request by MaxymVlasov**
Relates #710

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -599,7 +599,7 @@ Unlike most other hooks, this hook triggers once if there are any changed files
         - --hook-config=--use-standard-markers=true     # Boolean. Defaults to true (v1.93+), false (<v1.93). Set to true for compatibility with terraform-docs
     ```
 
-4. You can provide [any configuration available in `terraform-docs`](https://terraform-docs.io/user-guide/configuration/) as an argument to `terraform_doc` hook, for example:
+4. If you'd like to use `terraform-docs` config, you must to specify to hook it relative to git repo root path:
```
Comment (MaxymVlasov) [inline README.md:602]:
```suggestion
4. If you'd like to use `terraform-docs` config, you must to provide it relative to git repo root path to hook:
```

File: README.md [hunk 2]
```diff
@@ -599,7 +599,7 @@ Unlike most other hooks, this hook triggers once if there are any changed files
         - --hook-config=--use-standard-markers=true     # Boolean. Defaults to true (v1.93+), false (<v1.93). Set to true for compatibility with terraform-docs
     ```
 
-4. You can provide [any configuration available in `terraform-docs`](https://terraform-docs.io/user-guide/configuration/) as an argument to `terraform_doc` hook, for example:
+4. If you'd like to use `terraform-docs` config, you must to provide it relative to git repo root path to hook:
```
Comment (yermulnik) [inline README.md:602]:
```suggestion
4. If you'd like to use `terraform-docs` config file, you have to provide path to it, that is relative to git repo root path:
```

Comment (robinbowes) [reply README.md:602]: Suggested wording:
> If you want to use a `terraform-docs` config file, you must supply the path to the file, relative to the git repo root path:

Comment (yermulnik) [reply README.md:602]: @robinbowes Thanks. I adjusted my commit suggestion above.

Comment (MaxymVlasov) [reply README.md:602]: btw @robinbowes, I don't know can you use such feature in this repo, but if so, would be nice if you'll use suggestion block as George above - that drastically simplify apply of suggestions. Thank you
image

File: README.md [hunk 3]
```diff
@@ -610,7 +610,15 @@ Unlike most other hooks, this hook triggers once if there are any changed files
     > **Warning**  
     > Avoid use `recursive.enabled: true` in config file, that can cause unexpected behavior.
 
-5. If you need some exotic settings, it can be done too. I.e. this one generates HCL files:
+5. You can provide [any configuration available in `terraform-docs`](https://terraform-docs.io/user-guide/configuration/) as an argument to `terraform_docs` hook.
```
Comment (yermulnik) [inline README.md:613]:
```suggestion
5. You can provide [any configuration available in `terraform-docs`](https://terraform-docs.io/user-guide/configuration/) as an argument to `terraform_docs` hook:
```

**Review by yermulnik — APPROVED** [review]

**Review by yermulnik — APPROVED** [review]

**Comment by antonbabenko** [comment]
This PR is included in version 1.94.2 :tada:

**Pull request merged**